homeassistant/components/withings/__init__.py

```diff
@@ -116,37 +113,40 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.config_entries.async_update_entry(
             entry, data=new_data, options=new_options, unique_id=unique_id
         )
-    use_webhook = hass.data[DOMAIN][CONFIG][CONF_USE_WEBHOOK]
+    use_webhook = hass.data[DOMAIN][CONFIG].get(CONF_USE_WEBHOOK)
     if use_webhook is not None and use_webhook != entry.options[CONF_USE_WEBHOOK]:
         new_options = entry.options.copy()
         new_options |= {CONF_USE_WEBHOOK: use_webhook}
         hass.config_entries.async_update_entry(entry, options=new_options)
 
-    data_manager = await async_get_data_manager(hass, entry)
-
-    LOGGER.debug("Confirming %s is authenticated to withings", entry.title)
-    await data_manager.poll_data_update_coordinator.async_config_entry_first_refresh()
-
-    webhook.async_register(
-        hass,
-        const.DOMAIN,
-        "Withings notify",
-        data_manager.webhook_config.id,
-        async_webhook_handler,
+    client = ConfigEntryWithingsApi(
+        hass=hass,
+        config_entry=entry,
+        implementation=await config_entry_oauth2_flow.async_get_config_entry_implementation(
+            hass, entry
+        ),
     )
 
-    # Perform first webhook subscription check.
-    if data_manager.webhook_config.enabled:
-        data_manager.async_start_polling_webhook_subscriptions()
+    use_webhooks = entry.options[CONF_USE_WEBHOOK]
```

Comment:
You didn't change anything but I suppose this isn't optional in the config flow so it always comes back `True` or `False`

Reply:
The default is false, unless someone updated it via the options flow or has True in their yaml